### PR TITLE
DOCS-4050 Move adding user info content

### PR DESCRIPTION
**Pull request by kayayarai**
### What does this PR do?
Moves the content on adding user information to traces to a more findable place. https://docs-staging.datadoghq.com/kari/docs-4050/security_platform/application_security/add-user-info

### Motivation
Feedback from PMs, docs-4050

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by arnaudbreton** [comment]
@maxhedouin LGTM. Do you think it'd help with discovering this from the UI? 

Also, I think there is a link pointing to this page in the onboarding banner, right? 
image

**Review by arnaudbreton — APPROVED** [review]

**Review by maxhedouin — APPROVED** [review]

**Review by buraizu — APPROVED** [review]
Looks good, I'm approving, but just wanted to flag the casing in the further reading callouts. I recall a discussion about this where the consensus was to use sentence-style casing, for example:
`Add user information to traces`
I'll leave that up to you, but just wanted to bring it up.

File: content/en/security_platform/application_security/getting_started/dotnet.md [hunk 1]
```diff
@@ -5,6 +5,9 @@ code_lang: dotnet
 type: multi-code-lang
 code_lang_weight: 10
 further_reading:
+    - link: "/security_platform/application_security/add-user-info/"
+      tag: "Documentation"
+      text: "Adding User Information to Traces"
```
Comment (kayayarai) [inline content/en/security_platform/application_security/getting_started/dotnet.md:10]:
```suggestion
      text: "Adding user information to traces"
```

File: content/en/security_platform/application_security/getting_started/java.md [hunk 2]
```diff
@@ -5,6 +5,9 @@ code_lang: java
 type: multi-code-lang
 code_lang_weight: 0
 further_reading:
+- link: "/security_platform/application_security/add-user-info/"
+  tag: "Documentation"
+  text: "Adding User Information to Traces"
```
Comment (kayayarai) [inline content/en/security_platform/application_security/getting_started/java.md:10]:
```suggestion
  text: "Adding user information to traces"
```

File: content/en/security_platform/application_security/getting_started/nodejs.md [hunk 3]
```diff
@@ -5,6 +5,9 @@ code_lang: nodejs
 type: multi-code-lang
 code_lang_weight: 50
 further_reading:
+    - link: "/security_platform/application_security/add-user-info/"
+      tag: "Documentation"
+      text: "Adding User Information to Traces"
```
Comment (kayayarai) [inline content/en/security_platform/application_security/getting_started/nodejs.md:10]:
```suggestion
      text: "Adding user information to traces"
```

File: content/en/security_platform/application_security/getting_started/php.md [hunk 4]
```diff
@@ -5,6 +5,9 @@ code_lang: php
 type: multi-code-lang
 code_lang_weight: 40
 further_reading:
+    - link: "/security_platform/application_security/add-user-info/"
+      tag: "Documentation"
+      text: "Adding User Information to Traces"
```
Comment (kayayarai) [inline content/en/security_platform/application_security/getting_started/php.md:10]:
```suggestion
      text: "Adding user information to traces"
```

File: content/en/security_platform/application_security/getting_started/python.md [hunk 5]
```diff
@@ -5,6 +5,9 @@ code_lang: python
 type: multi-code-lang
 code_lang_weight: 50
 further_reading:
+    - link: "/security_platform/application_security/add-user-info/"
+      tag: "Documentation"
+      text: "Adding User Information to Traces"
```
Comment (kayayarai) [inline content/en/security_platform/application_security/getting_started/python.md:10]:
```suggestion
      text: "Adding user information to traces"
```

File: content/en/security_platform/application_security/getting_started/ruby.md [hunk 6]
```diff
@@ -5,6 +5,9 @@ code_lang: ruby
 type: multi-code-lang
 code_lang_weight: 30
 further_reading:
+    - link: "/security_platform/application_security/add-user-info/"
+      tag: "Documentation"
+      text: "Adding User Information to Traces"
```
Comment (kayayarai) [inline content/en/security_platform/application_security/getting_started/ruby.md:10]:
```suggestion
      text: "Adding user information to traces"
```

File: content/en/security_platform/application_security/setup_and_configure.md [hunk 7]
```diff
@@ -11,6 +11,9 @@ further_reading:
 - link: "/security_platform/default_rules/#cat-application-security"
   tag: "Documentation"
   text: "Out-of-the-Box Application Security Management Rules"
+- link: "/security_platform/application_security/add-user-info/"
+  tag: "Documentation"
+  text: "Adding User Information to Traces"
```
Comment (kayayarai) [inline content/en/security_platform/application_security/setup_and_configure.md:16]:
```suggestion
  text: "Adding user information to traces"
```

File: content/en/security_platform/application_security/add-user-info.md [hunk 8]
```diff
@@ -0,0 +1,271 @@
+---
+title: Tracking User Activity
+kind: documentation
+further_reading:
+- link: "/security_platform/application_security/"
+  tag: "Documentation"
+  text: "Protect against threats with Datadog Application Security Management"
+- link: "/security_platform/application_security/setup_and_configure/"
+  tag: "Documentation"
+  text: "Other Setup Considerations and Configuration Options"
```
Comment (kayayarai) [inline content/en/security_platform/application_security/add-user-info.md:10]:
```suggestion
  text: "Other setup considerations and configuration options"
```

**Pull request merged**